docs/faq/tricky_choices.md

```diff
@@ -0,0 +1,12 @@
+(tricky-choices)=
+# FAQ: Tricky Choices
+
+Inspired by IPLD's [tricky choices](https://ipld.io/design/tricky-choices/) section,
+this page describes some of the challenging decisions that could have gone multiple ways,
+and the considerations we weighed to arrive at the form we did. 
+
+## Arrays
+
+```{admonition} Hey
+Document me plz
+```
```

Comment:
@sneakers-the-rat Do you want to document our thinking for arrays in this PR? I can help write this but not until the end of the week, so maybe a separate PR would be best.

Reply:
Yeah lets do it later, I also wrote up my thoughts in our lab slack during the workshop bc I knew I would forget by the time it came to write the docs. Im itchin to get this merged so I can start on the next leg (no hurry tho!!!!!)